### PR TITLE
Removes the over-the-top smutty messages from PDA spam events.

### PR DESCRIPTION
**Pull request by OneOneThreeEight**
Feedback thread: https://forums.aurorastation.org/viewtopic.php?f=21&t=12268

Also replaces it with eridani augmentation advertisements. Might do more fluff-related stuff at a later date, but the current content being changed is really unacceptable given the rules against ERP and referencing erotic topics on the server.

Badkitty and wetskrell will stay, hold your horses.

### CODE REVIEW

**Comment by OneOneThreeEight** [comment]
Ignore whatever's going on in the additional commits, I'm too lazy and inept to use gitbash properly.

File: code/modules/events/money_spam.dm [hunk 1]
```diff
@@ -85,14 +85,14 @@
...
-					"Men of all species report AMAZING increases in length, width and stamina.",
-					"Discovery of this ONE Vaurca pheremone has led to years of genital growth progress! Sign on now!",\
-					"Is your sexlife growing dull? Your late-night sessions growing short? Have you considered Unathi testicle transplants? Greater stamina, greater virility, greater fun.")
+					sender = pick("Buy Dr. Augbaus",\
+					"Having biological troubles?")
+					message = pick("DR AUGBAUS: REAL Doctors, REAL Science, REAL Results!",\
+					"Dr. Augbaus was created by George Acuilar, M.D, a [current_map.boss_short] Certified Bio-mechanical Engineer who has treated over 70,000 patients sector wide with 'human problems'.",\
+					"After seven years of research, Dr Acuilar and his team came up with this simple breakthrough bio-aug enhancement formula.",\
+					"Men and women of all species report AMAZING increases in stamina, willpower and transcendent potential!",
+					"Discovery of this ONE Vaurca pheremone has led Eridans to have charge for their augs for weeks! Sign on now!",\
```
Comment (shedya) [inline code/modules/events/money_spam.dm]: Why would a pheromone do that? Typo in the word "pheromone", btw

Comment (OneOneThreeEight) [reply code/modules/events/money_spam.dm]: its snake oil, fam

File: code/datums/uplink/highly visible and dangerous weapons.dm [hunk 2]
```diff
@@ -112,4 +112,4 @@
 /datum/uplink_item/item/visible_weapons/custom_ka
 	name = "Kinetic Laser Assembly"
 	item_cost = 12
-	path = /obj/item/weapon/gun/custom_ka/frame01/illegal
\ No newline at end of file
+	path = /obj/item/weapon/gun/custom_ka/frame01/illegal
```
Comment (MarinaGryphon) [inline code/datums/uplink/highly visible and dangerous weapons.dm]: Mind checking this file out from master, if you know how? Need to remove this diff, if possible.

Comment (OneOneThreeEight) [reply code/datums/uplink/highly visible and dangerous weapons.dm]: Should be fixed.

**Comment by BurgerLUA** [comment]
I think a feedback thread might be needed. I find that many of these messages are pretty great.

**Comment by skull132** [comment]
> but the current content being changed is really unacceptable given the rules against ERP and referencing erotic topics on the server.

Nani kore wa. The implication of this would be we also ban dick jokes and poo jokes from being used ingame, IC, and OOC. Or any mature humor at all.

Though I'm not really opposed to more spam being _added_ that's !I C L O R E! friendly.

**Comment by OneOneThreeEight** [comment]
My issue is that referencing 'genitalia implants' is neither a tasteful or funny reference to 'mature humor' at all. I can understand if a joke is made to masturbation but it's much more subtle and clever, but "hey, unathi penis transplants, anyone?" is kind of outside the bounds of making fun of sexual organs or porn ads in real life.

I hope to not actually make this a slippery slope.

**Comment by MarinaGryphon** [comment]
Most of them are tasteful except for the one that there was really an issue with, `"Is your sexlife growing dull? Your late-night sessions growing short? Have you considered Unathi testicle transplants? Greater stamina, greater virility, greater fun."`
I still think replacing random spam with lore-related spam is good, though.

**Comment by OneOneThreeEight** [comment]
>What you're arguing for is the tastefulness of a joke. Which, while a valid argument to make, is not reinforced by the rules until it gets super bad. Pretense is important, as it decides how far we have to go with a specific change.

Hopefully this is the only occasion that we have to worry about this, then. I think it's important to draw specific lines where it's not okay to cross those in terms of decency's sake, but I also understand the importance of maintaining the integrity of most humor so that it doesn't turn into a tone-policing parade. It's an awkward balance in certain situations, but hopefully this is the only time where something gets withdrawn in favor of simply making better and more tasteful jokes.

**Comment by skull132** [comment]
I can't tell if you were agreeing or not. 🤔 

Anyways. `Having biological troubles?` doesn't make much sense. Since literally anything can be considered biological trouble.

**Comment by OneOneThreeEight** [comment]
https://forums.aurorastation.org/viewtopic.php?f=21&t=12268

**Comment by OneOneThreeEight** [comment]
> I can't tell if you were agreeing or not. 🤔
> 
> Anyways. `Having biological troubles?` doesn't make much sense. Since literally anything can be considered biological trouble.

It's a tentative, "eh, yes". The core argument that the PR forms a basis off of is that certain elements aren't appropriate and it'd be best to establish a little bit of merchandising in its place. I get the concern that it's in a way, censoring certain elements of 'mature humor' (which any one person can argue for or against that qualified definition all day), but taking into consideration that you're not allowed to write things about genitals for the in-game library, it seems only fair that the same standard gets applied in the game where it references such things distastefully.

**Pull request closed without merging**